bin/push_docker.js

```diff
@@ -51,20 +49,28 @@ const dependencies = {
 
 const configVersion = dependencies[configurationEntry].split('#')[1];
 const dockerRegistryDomain = 'quay.io';
-const repository = `${dockerRegistryDomain}/wire/webapp${suffix}`;
+const repository = `${dockerRegistryDomain}/wire/webapp${distributionParam ? `-${distributionParam}` : ''}`;
 
-const dockerImageTag = `${repository}:${pkg.version}-${commitShortSha}-${configVersion}${stage}`;
-const dockerImageStageTag = stageParam ? `${repository}:${stageParam}` : '';
+const tags = [];
+if (stageParam) {
+  tags.push(`${repository}:${stageParam}`);
+}
+if (releaseParam) {
+  tags.push(`${repository}:${releaseParam}-${configVersion}-${commitShortSha}`);
+}
 
 const dockerCommands = [
   `echo "$DOCKER_PASSWORD" | docker login --username "$DOCKER_USERNAME" --password-stdin ${dockerRegistryDomain}`,
-  `docker build . -t ${dockerImageTag} ${dockerImageStageTag ? `-t ${dockerImageStageTag}` : ''}`,
-  `docker push ${dockerImageTag}`,
+  `docker build . --tag ${commitShortSha}`,
 ];
 
-if (dockerImageStageTag) {
-  dockerCommands.push(`docker push ${dockerImageStageTag}`);
-}
+// prettier-ignore
```

Comment:
```suggestion
```

We don't accept `// prettier-ignore` statements in our PRs. 

[Prettier](https://prettier.io/), by its definition, is an opinionated code formatter and consequently it doesn't please everyone's style. That's okay because it has one big benefit: We don't lose time anymore by debating about coding styles in our projects and can invest the saved time into building new features and fixing existing bugs. Our projects also use auto formatting scripts, so that none of us wastes time indenting code. This helps us to focus on things that are really important by the end of the day. We are not going to start debating about styles and adding ignore rules here.

@lucendio, if you feel that Prettier's code styles don't feel "readable" and "natural", then please forward your concerns to [Prettier's issue tracker](https://github.com/prettier/prettier/issues).

Reply:
> We don't accept // prettier-ignore statements in our PRs.

If that is the case, How come that an overall amount of 32 `// prettier-ignore` can be found in this very repository?

Anyway, linter is now happy. We shoud be good to go.

Reply:
@lucendio, I think those `// prettier-ignore` statements come from a time where we didn't enforce our rules consistently. We strive to get better at that.

Thanks for resolving the Prettier issue here. This protects us from 32 becoming 33. 👍